subsys/bluetooth/controller/hal/nrf5/radio/radio_nrf5.h

```diff
@@ -5,12 +5,16 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#include "nrf_timer.h"
+#include <nrfx/hal/nrf_timer.h>
 
 #define HAL_RADIO_NS2US_CEIL(ns)  ((ns + 999)/1000)
 #define HAL_RADIO_NS2US_ROUND(ns) ((ns + 500)/1000)
 
-#define EVENT_TIMER NRF_TIMER0
+/* Use the timer instance ID, not NRF_TIMERx directly, so that it can be checked
+ * in radio_nrf5_ppi.h by the preprocessor.
+ */
+#define EVENT_TIMER_ID 0
+#define EVENT_TIMER    CONCAT(NRF_TIMER, EVENT_TIMER_ID)
```

Comment:
Why not use Zephyr's `_CONCAT()` instead? this one is only included if you compile for arm.
(This will break when targetting the nrf52_bsim)

Reply:
My bad. I wasn't aware that this is an architecture specific macro. Thanks for catching this!
Fixed.